### PR TITLE
Update Travis to run Go integration tests.

### PR DESCRIPTION
**Pull request by andyrzhao**
Also stops running Python integration tests.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-# Tweak for adding python3.5; see
-# https://github.com/travis-ci/travis-ci/issues/4794
-addons:
-  apt:
-    sources:
-      - deadsnakes
-    packages:
-      - python3.5
+script:
+  - go test integration/cli_test.go
```
Comment (shinfan) [inline .travis.yml]: EOF new line?

File: .travis.yml [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-  - cd python
-  - pip install tox
-  - pip install .
-script: tox -e $TOX_ENV
-after_success:
-  - if [[ "${TOX_ENV}" == "py27" ]]; then tox -e coveralls; fi
+  - GO111MODULE=on
+
+go:
+  - 1.12.x
```
Comment (shinfan) [inline .travis.yml]: Do we want to cover multiple versions?

File: .travis.yml (hunk 1 above)
Comment (andyrzhao) [reply .travis.yml]: Fixed.

File: .travis.yml (hunk 2 above)
Comment (andyrzhao) [reply .travis.yml]: Adding 1.11.x. 1.13.x is not yet available.

**Pull request merged**